src/traits/lowering-module.md

```diff
@@ -1,8 +1,8 @@
 # The lowering module in rustc
```

Comment:
I think this was all removed from rustc

Reply:
There's a WIP rustc PR that's going to add a new lowering module, we should add/fill this section again when that lands.